STM32/Temperature/Core/Src/Temperature.c

```diff
@@ -279,7 +278,9 @@ static void calibrateTypeInput(int noOfCalibrations, const CACalibration* calibr
 static void calibrateReadWrite(bool write)
 {
     if (write)
-        writeToFlashSafe(hcrc, 0, sizeof(portCalVal), (uint8_t *) portCalVal);
+    {
+        writeToFlashCRC(hcrc, (uint32_t) FLASH_ADDR_CAL, (uint8_t *) portCalVal, sizeof(portCalVal));
```

Comment:
On the oxygen board, you did this to print a message if an error occurs:

`if (writeToFlashCRC(hcrc_, (uint32_t) FLASH_ADDR_CAL, (uint8_t *) &cal, sizeof(cal)) != 0)
        {
            USBnprintf("Calibration was not stored in FLASH");
        }`